docusaurus.config.js

```diff
@@ -148,6 +148,42 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        // The application ID provided by Algolia
+        appId: '0ODKLVRPCZ',
+
+        // Public API key: it is safe to commit it
+        apiKey: 'cf5a5b2ee2c9278e23e79ac023bd5c7f',
+
+        indexName: 'condaorg',
+
+        // Optional: see doc section below
+        contextualSearch: true,
+
+        /* Optional: Specify domains where the navigation should occur through
+        window.location instead on history.push. Useful when our Algolia config crawls
+        multiple documentation sites and we want to navigate with window.location.href to them. */
+        externalUrlRegex: 'external\\.com|domain\\.com',
```

Comment:
This looks like an example value. Could we get rid of it or set this parameter to `undefined`?

Reply:
Good catch, thanks!